css/common/main.css

```diff
@@ -66,11 +72,23 @@ a:active {
   color: white;
 }
 
+.welcome-title {
+  margin: 1em 0;
+  text-align: center;
+  font-weight: 400;
+}
+
 .logo-img {
   width: 300px;
   margin: 2em;
 }
 
+.info-note {
+  text-align: center;
+  font-weight: 400;
+  font-size: 20px;
```

Comment:
Please read this answer to know about using px or rem for font-size: https://stackoverflow.com/a/11803273